migrations/102_fixtures_kyc.js

```diff
@@ -0,0 +1,60 @@
+require("babel-register");
+const getConfig = require("./config").getConfig;
+const getFixtureAccounts = require("./getFixtureAccounts").getFixtureAccounts;
+const getDeployerAccount = require("./config").getDeployerAccount;
+const promisify = require("../test/helpers/evmCommands").promisify;
+const toBytes32 = require("../test/helpers/constants").toBytes32;
+const identityClaims = require("../test/helpers/identityClaims").identityClaims;
+
+function serializeClaims(isVerified, isSophisticatedInvestor, hasBankAccount, isAccountFrozen) {
```

Comment:
looks cool, you could move it to `helpers/identityClaims` for other people to use